services/api/service.go

```diff
@@ -1068,27 +1171,28 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		}
 	}()
 
-	// Simulate the block submission and save to db
-	t := time.Now()
-	validationRequestPayload := &BuilderBlockValidationRequest{
-		BuilderSubmitBlockRequest: *payload,
-		RegisteredGasLimit:        slotDuty.GasLimit,
-	}
-	simErr = api.blockSimRateLimiter.send(req.Context(), validationRequestPayload, builderIsHighPrio)
-
-	if simErr != nil {
-		log = log.WithField("simErr", simErr.Error())
-		log.WithError(simErr).WithFields(logrus.Fields{
-			"duration":   time.Since(t).Seconds(),
-			"numWaiting": api.blockSimRateLimiter.currentCounter(),
-		}).Info("block validation failed")
-		api.RespondError(w, http.StatusBadRequest, simErr.Error())
-		return
+	// Construct simulation request.
+	opts := blockSimOptions{
+		ctx:      req.Context(),
+		highPrio: builderStatusCode == common.HighPrio,
```

Comment:
Consider changing to `builderStatusCode == common.HighPrio || builderStatusCode == common.Optimistic` and removing the line `opts.highPrio = true`.

Reply:
done!